lib/katello/plugin.rb

```diff
@@ -302,6 +303,370 @@
     multiple_actions_provider :hosts_change_content_source
   end
 
+  settings do
+    category :katello, N_('Content') do
```

Comment:
The `:katello` here doesn't seem to affect the category of the settings, which is something I don't really understand.. @ezr-ondrej

Reply:
What do you think by doesn't affect? It is internal category name, that users do not really see, so what have you expected it to affect?

Reply:
I expected it to affect the category here
```
[1] pry(main)> Setting.find_by(name: 'content_default_http_proxy')
=> #<Setting:0x000055ee680cb7a8
 id: 1,
 name: "content_default_http_proxy",
 value: nil,
 description: "Default HTTP Proxy for syncing content",
 category: "Setting",
 settings_type: nil,
 default: "--- \n...",
 created_at: Fri, 18 Mar 2022 18:10:51 UTC +00:00,
 updated_at: Tue, 19 Apr 2022 20:32:06 UTC +00:00,
 full_name: "Default HTTP proxy",
 encrypted: false>
```

Reply:
no, it does not, it is kept only in memory, as category field in the DB will be dropped and is already not considered for any User related purposes.